sjb/config/test_cases/ci-kubernetes-machine-api-operator.yml

```diff
@@ -0,0 +1,232 @@
+---
+parent: 'common/test_cases/minimal.yml'
+extensions:
+  actions:
+    - type: "host_script"
+      title: "Upload the default AWS credentiasl"
+      script: |-
+         ssh -F ${WORKSPACE}/.config/origin-ci-tool/inventory/.ssh_config openshiftdevel "mkdir ~/.aws"
+         scp -F ${WORKSPACE}/.config/origin-ci-tool/inventory/.ssh_config ~/.aws/credentials openshiftdevel:~/.aws
+    - type: "script"
+      title: "Install minikube"
+      script: |-
+        # Install minikube
+        curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.28.0/minikube-linux-amd64
+        chmod +x minikube
+        sudo mv minikube /usr/bin/
+        # Install kubectl
+        curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.10.0/bin/linux/amd64/kubectl
+        chmod +x kubectl
+        sudo mv kubectl /usr/bin/
+    - type: "script"
+      title: "Deploy kubernetes"
+      script: |-
+        sudo setenforce 0
+        # https://github.com/kubernetes/minikube/blob/master/docs/configuring_kubernetes.md
+        sudo minikube start --vm-driver=none --extra-config=kubelet.cgroup-driver=systemd
+    - type: "script"
+      title: "Build the machine api operator"
+      script: |-
+        # build the mao image
+        export GOPATH=/data
+        cd $GOPATH/src/github.com/openshift/machine-api-operator
+        go build -v -o bin/machine-api-operator github.com/openshift/machine-api-operator/cmd
```

Comment:
Why not `make build NO_DOCKER=1` ?

Reply:
Now as we have https://github.com/openshift/machine-api-operator/pull/27 merged, we can simplify it into sth like:
```sh
export GOPATH=/data
cd $GOPATH/src/github.com/openshift/machine-api-operator
go get -u github.com/openshift/imagebuilder/cmd/imagebuilder
sudo mv /data/bin/imagebuilder /usr/bin
make image NO_DOCKER=1
```